drivers/gles3/storage/material_storage.cpp

```diff
@@ -2947,6 +2947,18 @@ void MaterialStorage::material_update_dependency(RID p_material, DependencyTrack
 	}
 }
 
+LocalVector<ShaderGLES3::TextureUniformData> get_texture_uniform_data(const Vector<ShaderCompiler::GeneratedCode::Texture> &texture_uniforms) {
+	LocalVector<ShaderGLES3::TextureUniformData> texture_uniform_data;
+	for (int i = 0; i < texture_uniforms.size(); i++) {
+		auto num_textures = texture_uniforms[i].array_size;
```

Comment:
```suggestion
		int num_textures = texture_uniforms[i].array_size;
```

As a matter of style, we don't use ``auto`` in the godot codebase